server/game/core/Player.js

```diff
@@ -424,6 +443,21 @@ class Player extends GameObject {
         return null;
     }
 
+
+    /**
+     * Returns ths top cards of the player's deck
+     * @returns {import('./card/CardTypes').PlayableCard[]} the Card,© or null if the deck is empty
+     */
+    getTopCardsOfDeck(numCard = 1) {
```

Comment:
Do you think the default adds value here? I feel like this form would mostly be used when the value is something other than 1